dags/operators/moz_dataproc_operator.py

```diff
@@ -91,6 +91,9 @@ class DataprocClusterCreateOperator(DataprocOperationBaseOperator):
     :type master_disk_type: str
     :param master_disk_size: Disk size for the master node
     :type master_disk_size: int
+    :param master_num_local_ssds : Number of local SSDs to mount
```

Comment:
Please add a more descriptive comment on why we would ever want to set these. And what a defualt of 0 means

Reply:
Added better doc strings: e04d7375